src/nestjs-back/src/users/users.controller.ts

```diff
@@ -30,19 +30,18 @@ import { editFileName, imageFileFilter } from 'src/utils/upload';
 import { ValidateTfaDto } from './dto/validate-tfa-dto';
 import { IsLoggedInUserGuard } from "./guard/is-logged-in-user.guard";
 import { JwtAuthGuard } from "../auth/guard/jwt-auth.guard";
-import { QueryFailedError } from 'typeorm';
 
 @Controller('users')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
-  @UseGuards(JwtAuthGuard)
+  // @UseGuards(JwtAuthGuard)
```

Comment:
Can you explain why this shouldn't require authentication?

Reply:
Just saw this and fixed! This was debug